swag/client/client.py

```diff
@@ -74,117 +85,182 @@ async def process(self, message):
         try:
             if message.content.startswith("!$wagdmin"):
                 await execute_swagdmin_command(self, message)
-            elif message.content.startswith("!$wag"):
-                await execute_swag_command(self, message)
-            elif message.content.startswith("!$tyle"):
-                await execute_style_command(self, message)
-            elif message.content.startswith("!€agnotte"):
-                await execute_cagnotte_command(self, message)
-#            elif message.content.startswith("!Yfu"):
-#                await execute_yfu_command(self, message)
-        except NotEnoughSwagInBalance:
+        except InvalidTimeZone as e:
             await message.channel.send(
```

Comment:
We still have to send a message even we /commands?

Reply:
I just didn't code /commands for !$wagdmin commands. It's because for the moment I didn't find a way to make those commands visible only for specific user

Reply:
Ok, sorry, didn't catch that it was swagdmin commands